packages/gatsby-theme-project-portal/src/layouts/SearchPageLayout.stories.tsx

```diff
@@ -0,0 +1,108 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import { SearchPageLayout } from "./SearchPageLayout"
+import * as ProjectTeamStories from "../components/ProjectTeam.stories"
+
+const meta: Meta<typeof SearchPageLayout> = {
+  component: SearchPageLayout,
+  tags: ["autodocs"],
+}
+
+export default meta
+
+type Story = StoryObj<typeof SearchPageLayout>
+
+export const Primary: Story = {
+  args: {
+    data: {
+      allGeneralPage: {
+        nodes: {
+          slug: "/about",
+          lede: "About the Tests",
+          aims: [
+            {
+              title: "About Aim",
+              text: "Some text",
+            },
+          ],
+          faq: [
+            {
+              title: "Is this an FAQ question?",
+              text: "Yes, and here is the answer.",
+            },
+          ],
+          title: null,
+        },
+      },
+      allProject: {
+        nodes: {
```

Comment:
You could consider reusing the data from other stories here. You can do something like this:
```typescript
import * as AboutPageStories from "./AboutPageLayout.stories"

...

export const OneGeneralPage: Story = {
  args: {
    data: {
      allGeneralPage: {
        nodes: [
          {
            slug: "about",
            ...AboutPageStories.Primary.args.data.generalPage,
          },
        ],
      },
      allProject: {
        nodes: [],
      },
      site: { siteMetadata: { siteUrl: "https://localhost" } },
    },
  },
}
```